resources/js/store/modules/user/getters.js

```diff
@@ -2,5 +2,6 @@
 
 export default {
   user: state => state.user,
+  userGroup: state => state.user.group,
```

Comment:
this variable seems redundant, as the backend will add the user group onto the user object